gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -458,7 +486,7 @@ protected void setup(Context context) {
             this.fs, SequenceFileOutputFormat.getOutputPath(context).toUri().getPath(), TaskState.class);
       } catch (IOException ioe) {
         LOG.error("Failed to setup the mapper task", ioe);
-        return;
+        throw new RuntimeException(ioe);
```

Comment:
Why not replace above two lines with:

`throw new RuntimException("Failed to setup the mapper task", ioe);`